# HG changeset patch
# User Brian <runningnak3d@gmail.com>
# Date 1277924563 21600
# Branch trunk
# Node ID e93c431961731f1bf191e6dd2e0fff52a8a61405
# Parent  61015aa420a7ff06f635b2782966cf1181a7384c
* Some update to chat enums
* Thanks TOM_RUS

diff -r 61015aa420a7 -r e93c43196173 src/server/game/Entities/Player/SocialMgr.h
--- a/src/server/game/Entities/Player/SocialMgr.h	Wed Jun 30 12:58:00 2010 -0600
+++ b/src/server/game/Entities/Player/SocialMgr.h	Wed Jun 30 13:02:43 2010 -0600
@@ -43,7 +43,8 @@
 {
     SOCIAL_FLAG_FRIEND      = 0x01,
     SOCIAL_FLAG_IGNORED     = 0x02,
-    SOCIAL_FLAG_MUTED       = 0x04                          // guessed
+    SOCIAL_FLAG_MUTED       = 0x04,                          // guessed
+    SOCAIL_FLAG_RAF         = 0x08                           // Recruit A Friend
 };
 
 struct FriendInfo
@@ -112,7 +113,7 @@
 };
 
 #define SOCIALMGR_FRIEND_LIMIT  50
-#define SOCIALMGR_IGNORE_LIMIT  25
+#define SOCIALMGR_IGNORE_LIMIT  50
 
 class PlayerSocial
 {
